internals/daemon/response.go

```diff
@@ -169,22 +168,30 @@ func (f fileResponse) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	http.ServeFile(w, r, string(f))
 }
 
+// Responsef builds an error Response that returns the status and formatted message.
+//
+// If no arguments are provided, formatting is disabled, and the format string
+// is used as is and not interpreted in any way.
+func ErrorResponse(status int, format string, v ...interface{}) Response {
```

Comment:
Are we sure we don't want this to have the `f` prefix? That's fairly typical in Go on these formatting functions. I don't have strong feelings either way, just want to make sure dropping it wasn't accidental.

Reply:
Thanks Ben. The final call was in favour of sticking with it this way. Thank you for checking!